ftplugin/go.vim

```diff
@@ -1 +1,2 @@
 autocmd BufWritePre <buffer> lua vim.lsp.buf.formatting_sync()
+autocmd BufWritePost <buffer> lua require('cstm.buffer.code_action').goimport()
```

Comment:
See why this isn't triggered on `InsertLeave`.

See: https://github.com/aliou/dotvim/blob/0dd8c651a4268f6bf885732fd234360cd2ec4332/config/files.vim#L13